src/_utils/index.ts

```diff
@@ -6,4 +8,11 @@ export const prefix = (
   return [].concat(mods.map(modifier => `${baseClass}-${modifier}`))
 }
 
+export const isTouchEventAvailable = () => {
+  if (canUseDOM) {
+    return 'ontouchstart' in window
+  }
+  return false
+}
```

Comment:
nit: `isTouchEventAvailable = () => canUseDom && 'ontouchstart' in window`

Reply:
Done